server/service.go

```diff
@@ -110,10 +122,15 @@ func (m *BoostService) StartHTTPServer() error {
 		IdleTimeout:       m.serverTimeouts.Idle,
 	}
 
+	// Start separate process to send validator preferences at regular interval.
+	go m.registerValidatorAtInterval(time.Second*384, m.done)
```

Comment:
I'm guessing this should be registerValidatorIntervalSec

Reply:
Missed that, thanks ! Fixed in e1d657247437af9b985129d2e6a1ec7f1b8891c6.